src/components/data/viewers/Toolbar.js

```diff
@@ -47,58 +59,74 @@ import {
     List as MetadataIcon,
     Refresh,
     Save,
-    Code,
+    Visibility as ReadOnlyIcon,
```

Comment:
This Visibility icon makes me think of something being hidden or not - like my password when filling out a form.  I couldn't find a better icon though to indicate Read Only 🤔 If there were a pencil with a slash through it, that might work but MUI doesn't appear to have one.